README.md

```diff
@@ -577,6 +577,21 @@ render() {
 }
 ```
 
+### Use of Default Geocoder to get address
+
```

Comment:
Sorry, I should've been more clear here. I meant adding the method to the table in docs here: https://github.com/react-native-maps/react-native-maps/blob/master/docs/mapview.md#methods

Reply:
Got it @christopherdro will do.